### PR TITLE
Replace `go.uber.org/atomic` and `github.com/pkg/errors` with standard library packages

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
No functionality changes. The packages are now forbidden by a linter rule.

### CODE REVIEW

File: pkg/sentry/csr/csr.go [hunk 1]
```diff
@@ -32,17 +30,17 @@ var oidSubjectAlternativeName = asn1.ObjectIdentifier{2, 5, 29, 17}
...
 	templ, err := genCSRTemplate(org)
 	if err != nil {
-		return nil, nil, errors.Wrap(err, "error generating csr template")
+		return nil, nil, fmt.Errorf("error generating csr template: %w", err)
 	}
 
 	csrBytes, err := x509.CreateCertificateRequest(rand.Reader, templ, key)
 	if err != nil {
-		return nil, nil, errors.Wrap(err, "failed to create CSR")
+		return nil, nil, fmt.Errorf("failed to create CSR: %w", err)
```
Comment (mukundansundar) [inline pkg/sentry/csr/csr.go:43]: nit: `csr` instead of `CSR`?

**Review by mukundansundar — APPROVED** [review]
lgtm.

**Review by berndverst — DISMISSED** [review]

File: pkg/components/middleware/http/registry.go [hunk 2]
```diff
@@ -62,11 +60,11 @@ func (p *Registry) Create(name, version string, metadata middleware.Metadata) (h
 	if method, ok := p.getMiddleware(name, version); ok {
 		mid, err := method(metadata)
 		if err != nil {
-			return nil, errors.Errorf("error creating HTTP middleware %s/%s: %s", name, version, err)
+			return nil, fmt.Errorf("error creating HTTP middleware %s/%s: %w", name, version, err)
 		}
 		return mid, nil
 	}
-	return nil, errors.Errorf("HTTP middleware %s/%s has not been registered", name, version)
+	return nil, fmt.Errorf("HTTP middleware %s/%s has not been registered", name, version)
```
[the remark this answers is not in the document]
Comment (ItalyPaleAle) [reply pkg/components/middleware/http/registry.go:67]: Linter is not complaining so 🤷

File: pkg/sentry/csr/csr.go (hunk 1 above)
Comment (ItalyPaleAle) [reply pkg/sentry/csr/csr.go:43]: CSR is an acronym (Certificate Signing Request) so normally uppercase is correct.

**Comment by yaron2** [comment]
@ItalyPaleAle please check the e2e tests failing.

**Comment by ItalyPaleAle** [comment]
@yaron2 fixed

**Review by mukundansundar — APPROVED** [review]
lgtm

**Pull request merged**